src/Discord.Net.Rest/Entities/AuditLogs/DataTypes/ScheduledEventInfo.cs

```diff
@@ -50,7 +50,7 @@ public class ScheduledEventInfo
         /// <summary>
         ///     Gets the metadata for the entity associated with the event.
         /// </summary>
-        public string Location { get; }
+        public Optional<string>? Location { get; }
```

Comment:
Optional should not be nullable

Reply:
@Rekkonnect any further comments?

Reply:
`Optional<T>` and by consequence, `Optional<string>` on its own represents 3 states:
- `HasValue: false`: the object is not null, and denotes that there is no value of `string`
- `HasValue: true`: the object is not null, and denotes that there is a value of `string`, which is either `null` or not `null`.

What you do is introduce a fourth state, which you treat the exact same way as `HasValue: false`.

Sorry for the delayed response, I forgot

Reply:
you are actually 100% correct. ive addressed the optional situation, but the ternary expressions cannot really be improved as far as i can tell.